pkg/kubectl/cmd/patch.go

```diff
@@ -22,10 +22,10 @@ import (
 	"reflect"
 	"strings"
 
-	jsonpatch "github.com/evanphx/json-patch"
 	"github.com/golang/glog"
 	"github.com/spf13/cobra"
 
+	jsonpatch "github.com/evanphx/json-patch"
```

Comment:
That's actually unnecessary move, but I'll blame your goland :wink:

Reply:
Ah, will fix :)